### PR TITLE
Mute errors in parsing class definitions

### PR DESCRIPTION
**Pull request by samuelli**
- [ ] CHANGELOG.md has been updated

Testing this out for a hydrolysis 1.25.0 release

### CODE REVIEW

**Comment by rictic** [comment]
Do you have more detail with this change? What was it doing before and how does this PR change that?

**Comment by samuelli** [comment]
When using NPM hydrolysis 1.24.1, an exception occurred when analysing repos that targeted Polymer 2.0, including hybrid elements which default to 2.0.

This change simply prevents those exceptions from occurring and allows hybrid elements to be processed by hydrolysis. Justin thinks its where some of the Polymer definitions use getter syntax for `is`, instead of a string literal, which is what hydrolysis expects.

File: src/analyzer.ts [hunk 1]
```diff
@@ -317,7 +317,8 @@ export class Analyzer {
     for (const element of pseudoElements) {
       element.contentHref = href;
       this.elements.push(element);
-      this.elementsByTagName[element.is] = element;
+      if (element.is)
```
Comment (justinfagnani) [inline src/analyzer.ts]: add braces around then clause, here and below

**Comment by justinfagnani** [comment]
@rictic basically the class support in Hydrolysis was getting tripped up by 2.0 classes in core when analyzing hybrid elements.

File: src/analyzer.ts (hunk 1 above)
Comment (samuelli) [inline src/analyzer.ts]: Done.

**Comment by samuelli** [comment]
> @samuelli can you add a commit to bump the version to 1.25 so we can tag?

Done. I don't have to anything but `npm run build`, right?

**Comment by justinfagnani** [comment]
@samuelli pulling in those changes should be fine. Did you test in wc.org against master with these changes, or just 1.24.1 with them?

**Comment by justinfagnani** [comment]
I'm restarting the tests - it was a few browsers. Since we don't know they current status of the hydrolysis-1.x on Travis, it's hard to say if this broke anything, but I seriously doubt it.

**Comment by samuelli** [comment]
> @samuelli pulling in those changes should be fine. Did you test in wc.org against master with these changes, or just 1.24.1 with them?
> 
> edit: sorry, not master, but the hydrolysis-1.x branch

Tested wc.org locally with hydrolysis-1.x + changes. Hard to test extensively though. Once there's a tagged version I'll be able to deploy it to staging and see more extensive results there.

**Comment by justinfagnani** [comment]
Ok, some browsers still failing, but they probably already were. Merging.

**Pull request merged**